daos/1.json

```diff
@@ -0,0 +1 @@
+{"txHash":"0x233aaaf46f21486a660690c5ddff8a5444758957ba4273cf19cad4807f046b76","daoAddress":"0xd6283720Ce0a1c194DD830F31A01b898C2Fa038b","apps":[{"id":"0xceebe05dfa0c97985abadd7e19e480ac41793e500635eafd3a7d14a65be7f2df","proxy":"0x827dbC2E20141AA0C555f02d051E5462b92773A1"},{"id":"0x328cec85eddcd3372aaff5f6d4d71738294e0c0fccb0c33594e209d8032333d8","proxy":"0xA5bd326080EBD21eDDE22bcB723dDfe9f6C4A11e"}]}
```

Comment:
I guess we should not commit those. and add tha `daos` directory to the `.gitignore` file